Resources/public/js/views/services/ez-locationviewviewservice.js

```diff
@@ -567,6 +568,60 @@ YUI.add('ez-locationviewviewservice', function (Y) {
             this.set('languageCode', this.get('request').params.languageCode);
         },
 
+        /**
+         * Removes translation form content.
+         *
+         * @method _deleteTranslation
+         * @protected
+         * @param {String} languageCode
+         */
+        _deleteTranslation: function (languageCode) {
+            var content = this.get('content'),
+                options = {
+                    api: this.get('capi')
+                };
+
+            content.removeTranslation(options, languageCode, Y.bind(function() {
+                this._notify(
```

Comment:
I don't see any error handling for remove translation or I missed that. This assumes it will be always the success.